assets/training/model_management/environments/foundation-model-inference/context/Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN apt install git -y
 
 # Install pip for Python 3.10
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends wget && \
```

Comment:
Wondering if theres any CI pipeline running there to verify the change in this repo?